packages/react-aria/src/hooks/useAriaButton.test.ts

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { useARIAButton } from './useAriaButton';
+import { renderHook } from '@testing-library/react-hooks';
+import { ObjectShorthandProps } from '@fluentui/react-utilities';
+
+describe('useARIAButton', () => {
```

Comment:
I think we also need tests to know that the handlers are preventing default and stopping propagation correctly

Reply:
Also, would need tests to ensure keyboard handling